### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by alissonlauffer**

### CODE REVIEW

**Review by iiiiii1wepfj — APPROVED** [review]

File: locales/pt-BR/youtube.json [hunk 1]
```diff
@@ -5,5 +5,6 @@
   "ytdl_video_button": "🎬 Vídeo",
   "ytdl_file_too_big": "Desculpe! Não posso baixar esta mídia porque ela excede o meu limite de upload de 200MB.",
   "ytdl_send_error": "Desculpe! Não foi possível enviar o vídeo devido a um erro.\n<b>Erro:</b> <code>{errmsg}</code>",
-  "ytdl_downloading": "Baixando..."
+  "ytdl_downloading": "Baixando...",
+  "ytdl_button_denied": "Este botão não é para você."
```
Comment (AndrielFR) [inline locales/pt-BR/youtube.json:9]: Duplo sentido haha.

**Review by AndrielFR — APPROVED** [review]

File: locales/pt-BR/youtube.json (hunk 1 above)
Comment (iiiiii1wepfj) [reply locales/pt-BR/youtube.json:9]: what?

Comment (alissonlauffer) [reply locales/pt-BR/youtube.json:9]: analysis

Comment (AndrielFR) [reply locales/pt-BR/youtube.json:9]: @iiiiii1wepfj Nothing. 😂

**Pull request merged**